packages/utils/src/crypto/packet/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { publicKeyCreate, publicKeyVerify } from 'secp256k1'
+import { randomFillSync } from 'crypto'
+import { PRIVATE_KEY_LENGTH } from './constants'
+
+/**
+ * Generates a secp256k1 keypair used for an
+ * offline Diffie-Hellman key exchange.
+ * @returns a secp256k1 keypair
+ */
+export function generateKeyPair(): [privKey: Uint8Array, pubKey: Uint8Array] {
+  let privKey = new Uint8Array(PRIVATE_KEY_LENGTH)
+  let pubKey: Uint8Array
```

Comment:
Why not use the PublicKey type?

Reply:
the values are not really used as private key and public key

the name of the function is probably a bit misleading.

Reply:
please change then

Reply:
the function is now called sampleGroupElement()

Reply:
did you push these changes yet?